### ISSUE TITLE
sporadic error - Error in .cyFinally(res) : object 'res' not found

### ISSUE THREAD

**Issue by risserlin** [cytoscape/RCy3#186]
running pipelines from docker without fail eventually I will get the error: Error in .cyFinally(res) : object 'res' not found.  It doesn't matter which command I am running.  One minute RCy3 is connected and working and the next I keep getting the above error. 

sometimes, detach and re-attaching the RCy3 library will help fix it. 
sometimes, restarting cytoscape will fix it.
sometimes, restarting docker will fix it.  
And other I have to do all three of the above.  

I don't know where this issue is coming from but is it possible that there is something that is not be cleared or an empty res variable is triggering?

**Comment by risserlin** [comment]
uh oh! That means updating R on my computer.  Something I avoid like the plague.  Truthfully I didn't even think of doing that because I am so programmed to use docker.  I will see if that fixes it.

**Comment by risserlin** [comment]
I finally solved this issue.  this is not an RCy3 or a cytoscape issue.  This is a docker on mac issue.  I fixed it by downgrading my docker version to 4.5. There was another suggestion/recommendation but it required mac OS 11 and higher (which I don't have).  
Here is the thread that helped me just in case anyone else has this issue when running pipelines in docker that use cytoscape and RCy3 - https://github.com/docker/for-mac/issues/3448
